### PR TITLE
fix(staff): Remove incorrrect u2f error message

### PR DESCRIPTION
**Pull request by schew2381**
Hopefully this fixes the random u2f error msg popping up when it shouldn't be

### CODE REVIEW

File: static/app/components/superuserStaffAccessForm.tsx [hunk 1]
```diff
@@ -59,7 +59,9 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
       return;
     }
 
-    await this.getAuthenticators();
+    const authenticators = await this.getAuthenticators();
+    this.setState({authenticators: authenticators});
+
     // Set the error state if there are no authenticators and U2F is on
     if (!this.state.authenticators.length && !disableU2FForSUForm) {
```
Comment (dashed) [inline static/app/components/superuserStaffAccessForm.tsx:66]: `componentDidMount()` will only run once on mount, so this part needs to be updated as well:

```suggestion
    if (!authenticators && !disableU2FForSUForm) {
```

Comment (schew2381) [reply static/app/components/superuserStaffAccessForm.tsx:66]: wait do I need to keep the `.length` part?

Comment (dashed) [reply static/app/components/superuserStaffAccessForm.tsx:66]: @schew2381 ah yea that too.

Comment (schew2381) [reply static/app/components/superuserStaffAccessForm.tsx]: alright done

**Review by dashed — APPROVED** [review]

File: static/app/components/superuserStaffAccessForm.tsx [hunk 2]
```diff
@@ -59,9 +59,11 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
       return;
     }
 
-    await this.getAuthenticators();
+    const authenticators = await this.getAuthenticators();
+    this.setState({authenticators: authenticators});
```
Comment (sentaur-athena) [inline static/app/components/superuserStaffAccessForm.tsx:63]: I don't see this state being modified anywhere else. Does it even need to be state?

Comment (schew2381) [reply static/app/components/superuserStaffAccessForm.tsx:63]: Would removing it look like this? https://github.com/getsentry/sentry/pull/70365/

**Pull request merged**